tests/unit_test/llm_test.py

```
@@ -610,7 +611,7 @@ async def test_gpt3_faq_embedding_predict(self, mock_embedding, aioresponses):
                     {'id': test_content.vector_id, 'score': 0.80, "payload": {'content': test_content.data}}]}
             )
 
-            response, time_elapsed = await gpt3.predict(query, user=user, **k_faq_action_config)
+            response, time_elapsed = await gpt3.predict(query, user=user, collection=collection_name, **k_faq_action_config)
```

Ruff (0.15.7)

[warning] Unpacked variable `response` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)